### PR TITLE
RD-5110 The new `heal` workflow

### PR DESCRIPTION
**Pull request by tehasdf**
It's a bit more involved than it used to be - it's not about just
reinstalling anymore!

Also, I'm removing those images. The graph isn't needed there, and it's
bad enough that it makes the example harder to understand.

Instead, I simplify the blueprint a bit. No need for `database_host`.

### CODE REVIEW

**Comment by tehasdf** [comment]
staging https://docs.cloudify.co/staging/new-heal-doc-1/working_with/workflows/built-in-workflows/#the-heal-workflow

File: content/working_with/workflows/built-in-workflows.md [hunk 1]
```diff
@@ -217,86 +217,96 @@ This workflow simply calls the `stop` workflow, followed by `start`.
 
 **Workflow name:** **`heal`**
 
-**Workflow description:** Reinstalls the whole subgraph of the system topology by applying the `uninstall` and `install` workflows' logic respectively. The subgraph consists of all the node instances that are contained in the compute node instance which contains the failing node instance and/or the compute node instance itself. Additionally, this workflow handles unlinking and establishing all affected relationships in an appropriate order.
+**Workflow description:** Heals node-instances of a deployment, or a subset of them. Run the `check_status` and `heal` interfaces for each selected instance, and in case those operations fail or are not declared, reinstall the node instance. Instances that pass their `check_status` call are not healed or reinstalled.
```
Comment (geokala) [inline content/working_with/workflows/built-in-workflows.md:220]: 'Run' should probably be 'Runs' (as 'Heals' is there rather than 'Heal')

Comment (geokala) [reply content/working_with/workflows/built-in-workflows.md:220]: Same for 'reinstall' (should be 'reinstalls' to fit with 'Heals')

Comment (tehasdf) [reply content/working_with/workflows/built-in-workflows.md:220]: I prefer the `Heal` wording, then

**Review by geokala — APPROVED** [review]

**Review by glukhman — APPROVED** [review]

**Pull request merged**